api/src/main/java/org/apache/iceberg/ContentFile.java

```diff
@@ -165,6 +166,20 @@ default Long fileSequenceNumber() {
    */
   F copyWithoutStats();
 
+  /**
+   * Copies this file with only specific column stats. Manifest readers can reuse file instances;
+   * use this method to copy data and only copy specific stats when collecting files. If the
```

Comment:
should `If the columnsToKeepStats set is empty ...` be put in the `@param` below?